README.md

```diff
@@ -87,6 +96,22 @@ Note: you don't have to initialize submodule to run docker, dockerfile will set
 
 Note: If you want to build only one service, use `docker compose build <service-name>`, for example for backend it will be `docker compose build backend`.
 
+
+## Working with database
+
+1. You can run the application without containers, allowing you to develop things quickly. Refer to **Running the iNatAtor Application** above for instructions.
+
+2. You can verify connections are working by going to `localhost:8000/health` and `localhost:8000/echo_env`. Note that these paths are included right now to assist with rapid development, will be removed for production.
```

Comment:
1 and 2 are not actions, are just notes. I suggest keeping number only on actions.

Reply:
I will reword those, I think it's still important to verify connection as a step